docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -578,19 +578,24 @@ The default is 1000.
 ### nat-method
 
 ```bash tab="Syntax"
---nat-method=<METHOD>
-```
-
-```bash tab="Command Line"
---nat-method=upnp
+--nat-method=UPNP
 ```
 
 ```bash tab="Example Configuration File"
-nat-method="upnp"
+nat-method="UPNP"
 ```
 
-Specifies the method for handling [NAT environments](../../HowTo/Find-and-Connect/Using-UPnP.md). 
-Options are `upnp` and `none`. The default is `none` (that is, NAT functionality is disabled).
+Specify the method for handling [NAT environments](../../HowTo/Find-and-Connect/Using-UPnP.md). Options are: `UPNP` and `NONE`.
+The default is `NONE`, which disables NAT functionality.
+
+!!!tip
+    `UPNP` works well with a typical home or small office environment where a wireless router or modem provides NAT isolation. This should provide
```

Comment:
I think this tip could be updated to be the same as the one on the HowTo/Find-and-Connect/Using-UPnP page: 
UPnP support is often disabled by default in networking firmware. If disabled by default, explicitly enable UPnP support.

Reply:
Fixed.